Emacs 26: Emacs not response when press Enter in shell mode

alexei-28 opened the pull request:
Windows 10 (64 bit), Emacs 26.1

1. I download Emacs 26 from here : http://ftp.gnu.org/gnu/emacs/windows/emacs-26/emacs-26.1-x86_64.zip
2. Unpack and run pure Emacs (no external packages)
3. Connect android device to my computer
4. `M-x shell`
5. Run the next command `adb logcat -vtime`. Tool `logcat `- is a tool from Google to see android's device logging. It's very helpful.
6. This tool (`logcat`) generate many text interactively in shell.
7. So I press button `arrow up` and move cursor to the center of screen. 
8. Text is continues to be generated. 
9. And now I press button `Enter`
10. As result Emacs not response any more. Show progress bar forever. Nothing help. Only kill Emacs process from Task Manager.

Here screenshot:
![emacs_freeze](https://user-images.githubusercontent.com/303539/46243538-270b2d00-c3de-11e8-8ca9-8e390cee4d82.jpg)

Why this is happened?

snow8gaia commented:
there's no issue page or repo on github. so see https://github.com/pbatard/rufus/issues/1344

same for Emacs.

muffinmad committed:
Call vc-setup-buffer in vc-hg-log-incoming and vc-hg-log-outgoing

* lisp/vc/vc-hg.el (vc-hg-log-incoming, vc-hg-log-outgoing):
Call vc-setup-buffer.

dgutov committed:
Revert "Call vc-setup-buffer in vc-hg-log-incoming and vc-hg-log-outgoing"

This reverts commit 52e2682f015d70b0945d9037686eec5fb8761571.

alexei-28 commented:
After I reinstall Emacs , and install only few packages (remove unneeded packages) the  problem is gone.

eggert committed:
Port alternate signal stack to upcoming glibc 2.34

* src/sysdep.c (sigsegv_stack): Increase size to 64 KiB and align
it to max_align_t.  This copies from Gnulib’s c-stack.c, and works
around a portability bug in draft glibc 2.34, which no longer
defines SIGSTKSZ when _GNU_SOURCE is defined.

(cherry picked from commit f97e07ea807cc6d38774a3888a15091b20645ac6)
